src/java/org/apache/cassandra/repair/consistent/LocalSessions.java

```diff
@@ -353,13 +354,16 @@ public void cancelSession(TimeUUID sessionID, boolean force)
      */
     public synchronized void start()
     {
+        long startTime = nanoTime();
```

Comment:
this fails the tests

[junit-timeout] Property error detected:
[junit-timeout] Seed = 8335649466716534365
[junit-timeout] Examples = 10
[junit-timeout] Pure = false
[junit-timeout] Error: Rejecting access
[junit-timeout] Values:
[junit-timeout]         0 = accord.utils.DefaultRandom@5bd191c2
[junit-timeout] 
[junit-timeout] junit.framework.AssertionFailedError: Property error detected:
[junit-timeout] Seed = 8335649466716534365
[junit-timeout] Examples = 10
[junit-timeout] Pure = false
[junit-timeout] Error: Rejecting access
[junit-timeout] Values:
[junit-timeout]         0 = accord.utils.DefaultRandom@5bd191c2
[junit-timeout] 
[junit-timeout]         at accord.utils.Property$SingleBuilder.checkInternal(Property.java:242)
[junit-timeout]         at accord.utils.Property$SingleBuilder.check(Property.java:226)
[junit-timeout]         at accord.utils.Property$ForBuilder.check(Property.java:124)
[junit-timeout]         at org.apache.cassandra.repair.SlowMessageFuzzTest.slowMessages(SlowMessageFuzzTest.java:41)
[junit-timeout]         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[junit-timeout]         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[junit-timeout]         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[junit-timeout] Caused by: java.lang.IllegalStateException: Rejecting access
[junit-timeout]         at org.apache.cassandra.repair.FuzzTestBase$ClockAccess.checkAccess(FuzzTestBase.java:1477)
[junit-timeout]         at org.apache.cassandra.repair.FuzzTestBase$ClockAccess.nanoTime(FuzzTestBase.java:1423)
[junit-timeout]         at org.apache.cassandra.utils.Clock$Global.nanoTime(Clock.java:100)
[junit-timeout]         at org.apache.cassandra.repair.consistent.LocalSessions.start(LocalSessions.java:357)
[junit-timeout]         at org.apache.cassandra.service.ActiveRepairService.start(ActiveRepairService.java:265)
[junit-timeout]         at org.apache.cassandra.repair.FuzzTestBase$Cluster$Node.<init>(FuzzTestBase.java:1134)
[junit-timeout]         at org.apache.cassandra.repair.FuzzTestBase$Cluster.<init>(FuzzTestBase.java:753)
[junit-timeout]         at org.apache.cassandra.repair.SlowMessageFuzzTest.lambda$slowMessages$1(SlowMessageFuzzTest.java:43)
[junit-timeout]         at accord.utils.Property$SingleBuilder.checkInternal(Property.java:238)

Reply:
https://github.com/apache/cassandra/blob/trunk/test/unit/org/apache/cassandra/repair/FuzzTestBase.java#L1396-L1480